# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1471304364 14400
#      Mon Aug 15 19:39:24 2016 -0400
# Node ID c0cdf1c92086db3e9c2e9aad15fe19df66f5c109
# Parent  0448bf26fe98a30ab83f68b07c53458b593b61f0
strip header from op-kw-docs when generating libinterp/DOCSTRINGS

* libinterp/module.mk (libinterp/DOCSTRINGS): Strip header from
op-kw-docs file.

diff --git a/libinterp/module.mk b/libinterp/module.mk
--- a/libinterp/module.mk
+++ b/libinterp/module.mk
@@ -276,17 +276,17 @@ libinterp/dldfcn/PKG_ADD: $(DLDFCN_DEFUN
 	$(SHELL) $(srcdir)/libinterp/mk-pkg-add "$(srcdir)" $(DLDFCN_DEFUN_FILES) > $@-t && \
 	mv $@-t $@
 endif
 
 DOCSTRING_FILES += libinterp/DOCSTRINGS
 
 libinterp/DOCSTRINGS: $(LIBINTERP_DEFUN_FILES) libinterp/op-kw-docs | libinterp/$(octave_dirstamp)
 	$(AM_V_GEN)rm -f libinterp/DOCSTRINGS-t && \
-	( $(PERL) $(srcdir)/libinterp/gendoc.pl "$(srcdir)" $(LIBINTERP_DEFUN_FILES); cat $(srcdir)/libinterp/op-kw-docs ) > libinterp/DOCSTRINGS-t && \
+	( $(PERL) $(srcdir)/libinterp/gendoc.pl "$(srcdir)" $(LIBINTERP_DEFUN_FILES); $(SED) -ne '/^\x1d/,$$p' $(srcdir)/libinterp/op-kw-docs ) > libinterp/DOCSTRINGS-t && \
 	$(call move_if_change_rule,libinterp/DOCSTRINGS-t,$@)
 
 OCTAVE_INTERPRETER_TARGETS += \
   $(OCT_FILES) \
   $(DLDFCN_PKG_ADD_FILE)
 
 DIRSTAMP_FILES += libinterp/$(octave_dirstamp)
 
